javascript/javascript1/week1/housey-pricey.js

```diff
@@ -0,0 +1,21 @@
+//housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+//let width
+//let depth
+//let height
+//let volumeInMeters = width * depth * height
+//function calculateHousePrice(volumeInMeters, gardenSizeInM2) {
+//  return
+//}
+
+let volumeInMetersPeter = 8 * 10 * 10;
+let gardenSizeInM2Peter = 100;
+let volumeInMetersJulia = 5 * 11 * 8;
+let gardenSizeInM2Julia = 70;
+let housePricePeter = volumeInMetersPeter * 2.5 * 1000 + gardenSizeInM2Peter * 300;
+let housePriceJulia = volumeInMetersJulia * 2.5 * 1000 + gardenSizeInM2Julia * 300;
+let houseCostPeter = 2500000;
+let houseCostJulia = 1000000;
+if (houseCostPeter > housePricePeter) { console.log("Peter is paying too much") }
+else if (houseCostPeter === housePricePeter) { console.log("Peter is paying a good price") }
+else { console.log("Peter is paying too little") }
+if (houseCostJulia > housePriceJulia) { console.log("Julia is paying too much") } else if (houseCostJulia === housePriceJulia) { console.log("Julia is paying a good price")} else { console.log("Julia is paying too little")}
```

Comment:
Please use Prettier for formatting your code.

Reply:
Prettier doesn't work for me... I have updated VS code to the latest version.. Maybe you will have some time on Sunday to help with it?

Reply:
Please ask one of the mentors who will be there on Sunday to check why Prettier doesn't work for you. You can also check with your classmates.